src/fn/upload-custom-translations.js

```diff
@@ -81,6 +114,76 @@ function parse(filePath, options) {
   });
 }
 
+function checkTranslations(translations, lang, templatePlaceholders) {
+  let mf = null;
+  try {
+    mf = new MessageFormat(lang);
+  } catch (e) {
+    log.warn(`Cannot check '${lang}' translations: ${e.message}`);
+  }
+  const placeholders = extractPlaceholdersFromTranslations(translations);
+  let formatErrorsFound = 0,  placeholderErrorsFound = 0, emptiesFound = 0;
+  for (const [msgKey, msgSrc] of Object.entries(translations)) {
+    if (!msgSrc) {
+      emptiesFound++;
+    } else if (/{{[\s\w.#^/'|]+}}/.test(msgSrc)) {
```

Comment:
Pull this regex out as a constant at the top of the file so it can be named for readability.

Reply:
Done